ui/src/features/inference/stream/stream.tsx

```diff
@@ -12,31 +12,22 @@ const useSetTargetSizeBasedOnVideo = (
 ) => {
     useEffect(() => {
         const video = videoRef.current;
+        if (!video) return;
 
-        const onsize = video?.addEventListener('loadedmetadata', (event) => {
-            const target = event.currentTarget as HTMLVideoElement;
+        const onLoaded = () => {
+            if (video.videoWidth && video.videoHeight) setSize({ width: video.videoWidth, height: video.videoHeight });
```

Comment:
It’s better to use curly braces, otherwise there’s a risk that the content could be moved to the next line and not execute as expected

```suggestion
            if (video.videoWidth && video.videoHeight) {
                setSize({ width: video. videoWidth, height: video.videoHeight })
            }
```

Reply:
Totally agree, updated